services/crypto_create.proto

```diff
@@ -151,4 +151,18 @@ message CryptoCreateTransactionBody {
      * If true, the account declines receiving a staking reward. The default value is false.
      */
     bool decline_reward = 17;
+
+    /**
+     * The bytes to be used as the account's alias. The bytes will be 1 of 2 options. 
+     * It will be the serialization of a protobuf Key message for an ED25519/ECDSA primitive key type. 
```

Comment:
There are two ECDSA key types in the `Key` proto. Should adjust the other references too.
```suggestion
     * It will be the serialization of a protobuf Key message for an ED25519/ECDSA_SECP256K1 primitive key type. 
```

Reply:
Fixed